include/wimlib/list.h

```diff
@@ -212,17 +212,25 @@ list_splice_tail(struct list_head *list, struct list_head *head)
  * @pos:	the type * to cursor
  * @member:	the name of the list_struct within the struct.
  */
-#define list_next_entry(pos, member) \
+#ifdef _MSC_VER
```

Comment:
No need for any of these redefs, except one, if using `/std:clatest` with VS2022.